sentry.gradle

```diff
@@ -92,7 +92,10 @@ gradle.projectsEvaluated {
               } catch (FileNotFoundException e) {
                   project.logger.info("file not found '$propertiesFile' for '$variant'")
               }
-              def cliExecutable = sentryProps.get("cli.executable", "$reactRoot/node_modules/@sentry/cli/bin/sentry-cli")
+
+              def resolvedCliPackage = new File(["node", "--print", "require.resolve('@sentry/cli/package.json')"].execute(null, rootDir).text.trim()).getParentFile();
```

Comment:
```suggestion
              def resolvedCliPackage = null;
              try {
              resolvedCliPackage = new File(["node", "--print", "require.resolve('@sentry/cli/package.json')"].execute(null, rootDir).text.trim()).getParentFile();
              } catch (Throwable ignored) {}
```

Reply:
this could throw in a few different ways, ideally we do try catch and keep running in case it goes wrong

Reply:
`getParentFile` returns null when the file doesn't exist, but no harm in checking it too!